pkgs/top-level/all-packages.nix

```diff
@@ -22113,7 +22113,9 @@ with pkgs;
   inherit (callPackages ../servers/unifi { })
     unifiLTS
     unifi5
-    unifi6;
+    unifi6
+    unifi7;
+
   unifi = unifi6;
```

Comment:
is unifi7 stable enougth to be default?

Reply:
they describe it as a release candidate, so better to not switch over quite yet